drivers/include/io1_xplained.h

```diff
@@ -29,77 +29,72 @@ extern "C" {
 #endif
 
 /**
- * @brief Device descriptor for the IO1 Xplained extension.
+ * @brief IO1 Xplained driver return codes
```

Comment:
`@name` is only for groups (marked by an `@{`-`@}`-pair). This is a type, so `@brief` ;-)